src/figuresClasses.ts

```diff
@@ -1,11 +1,85 @@
-export interface Figure {}
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: string;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(
```

Comment:
The constructor parameters are being implicitly assigned to the class properties due to the use of access modifiers. You can remove the explicit assignments to `this.sideA`, `this.sideB`, and `this.sideC` inside the constructor.